### DISCUSSION TITLE
Related object fields

### DISCUSSION THREAD

**Discussion by mmattm** [VirtusLab/strapi-plugin-navigation#261] [Q&A]
Hi! 

Is it possible to hide specific fields of the "related" object. 

For instance i have a collection type "Page" and I don't want to load all contents & texts of these pages everytime i render my navigation component in my frontend.

**Comment by cyp3rius** [comment]
Hello @mmattm,

as for now it's not possible but idea is worth to consider 👍  Anyway it's not going to be a trivial like `fields` query parameter in Strapi, because each navigation item might be related to different Collection or Single Type. By the first glance it will need an UI configuration zone on the Settings page.

**Comment by QuarkAgency** [comment]
Hello, any news about that ?
Something like add a field to select which field to be populated or which field to exclude here :

![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/117939567/eb935b7e-2aa9-4f7d-a95a-f32070e41088)
